src/Roslyn.Suppressions/Roslyn.Suppressions/InProcess/RoslynSettingsFileSynchronizer.cs

```diff
@@ -122,13 +122,12 @@ public async Task UpdateFileStorageAsync()
                     Debug.Assert(fullSolutionFilePath != null, "Not expecting the solution name to be null in Connected Mode");
                     var solnNameWithoutExtension = System.IO.Path.GetFileNameWithoutExtension(fullSolutionFilePath);
 
-                    var allSuppressedIssues = serverIssuesStore.Get();
+                    var allSuppressedIssues = serverIssuesStore.Get().Where(x => x.IsResolved);
```

Comment:
here's a logic change I think this change requires some tests.

Reply:
It's covered by the existing tests. I just moved this LINQ call from one place to the other.